main/install/configuration.dist.php

```diff
@@ -283,7 +283,9 @@
 // $_configuration['allow_course_introduction_low_security'] = false;
 // Prevents the duplicate upload in assignments
 // $_configuration['assignment_prevent_duplicate_upload'] = false;
-// Set ConsideredWorkingTime work extra field variable from main/admin/extra_fields.php?type=work
+//Show student progress in My courses page
+//$_configuration['my_course_progress'] = false;
+// Set ConsideredWorkingTime work extra field variable to show in MyStudents page works report
```

Comment:
Not sure if this is the result of a merge, but you shouldn't change the comments for another feature in your PR. This confuses things.